### PR TITLE
Makes Houndmaster/Hornet Not Shite

### PR DESCRIPTION
**Pull request by Dark1Star3s**
Scoped Accuracy increased by 2
Fire Delay from 3 seconds to 1.2
AP from 30 to 45.
Shrapnel Chance to 1.1 from 1.0
Arterial chance to 1.3 from 1.0
Window/Wall Pen from 0 to 2.

### CODE REVIEW

**Review by Kaksisilma — APPROVED** [review]
Looks OK.

File: code/modules/projectiles/projectile/bullets.dm [hunk 1]
```diff
@@ -219,11 +219,11 @@
 /obj/item/projectile/bullet/rifle/military/large
 	fire_sound = 'sound/weapons/gunshot/hornet.ogg'
 	damage = 45
-	armor_penetration = 30
-	penetrating = 0
+	armor_penetration = 45
+	penetrating = 2
```
Comment (ProbablyCarl) [inline code/modules/projectiles/projectile/bullets.dm:223]: It'll go through walls.

Comment (Vizoplez) [reply code/modules/projectiles/projectile/bullets.dm:223]: Window/Wall Pen from 0 to 2.
intended by PR, apparently.
It's a 12.7mm rifle anyway, so its kinda monkaw that it doesn't.

Comment (Dark1Star3s) [reply code/modules/projectiles/projectile/bullets.dm:223]: > It'll go through walls.

Bullpups and Bulldogs also go through walls- in fact most ballistic ARs do.

Comment (Yacobpo157) [reply code/modules/projectiles/projectile/bullets.dm:223]: A DMR in a round large than any current ballistic AR should also penetrate walls.

**Comment by ProbablyCarl** [comment]
I'll be merging this shortly.
I did a stupid and misinterpreted the penetration stuff, confusing it with something else while tired.

**Pull request merged**